src/models/room.ts

```diff
@@ -2039,20 +2039,41 @@ export class Room extends EventEmitter {
         // -1 because these numbers include the syncing user
         const inviteJoinCount = joinedMemberCount + invitedMemberCount - 1;
 
+        // get service members (e.g. helper bots) for exclusion
+        let excludedUserIds: string[] = [];
+        const mFunctionalMembers = this.currentState.getStateEvents(EventType.RoomFunctionalMembers, "");
+        if (mFunctionalMembers && mFunctionalMembers.getContent() && Array.isArray(mFunctionalMembers.getContent().service_members)) {
```

Comment:
Just FYI, you have access to TS here. And getContent is never falsey

```suggestion
        if (Array.isArray(mFunctionalMembers?.getContent().service_members)) {
```

Reply:
I blame line 2019 for bad influence.
https://github.com/matrix-org/matrix-js-sdk/blob/dc90115a1bbd733351097fc97d93f47b0e31fa8d/src/models/room.ts#L2019

Reply:
Oh for sure, the TS conversion I did definitely did not exhaustively improve every single possible line, but rather focused on types.